### DISCUSSION TITLE
Should javascript reference be kept up to date and in that case how?

### DISCUSSION THREAD

**Discussion by flygarhunden** [usebruno/bruno#2746] [General]
I upgraded and read release notes and saw some new functions being added manipulating variables https://github.com/usebruno/bruno/pull/2531. I went to https://docs.usebruno.com/scripting/javascript-reference to read about the functions but did not find them there.

Hence, my question is that page meant as a complete reference? Maybe consider generated documentation in that case to avoid mismatch? After googling seems javascript does not a standard for that like java that I am more familiar with so maybe not so easy?

**Comment by jwetzell** [comment]
The documentation in general is pretty behind and incomplete. JSDoc would be the JavaDoc equivalent here.
